### PR TITLE
Adding support file to handle questions around support issues for InSpec

### PR DESCRIPTION
**Pull request by kekaichinose**
Signed-off-by: Keka Ichinose <rikoomi@hotmail.com>

## Related Issue

ref: https://github.com/inspec/inspec/issues/3594
ref: https://github.com/inspec/inspec/issues/4315

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New content (non-breaking change)
- [ ] Breaking change (a content change which would break existing functionality or processes)

## Checklist:

- [X] I have read the **CONTRIBUTING** document.

### CODE REVIEW

**Review by miah — APPROVED** [review]

File: SUPPORT.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+We use GitHub issues to track bugs and feature requests. If you need support help (i.e. with using InSpec), we recommend that if you are a:
+
+* Chef customer
```
Comment (mjingle) [inline SUPPORT.md]:
```suggestion
* Chef customer:
```

File: SUPPORT.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  * Please reach out to your Customer Success manager and they can assist you
+* Chef community member
```
Comment (mjingle) [inline SUPPORT.md]:
```suggestion
* Chef community member:
```

File: SUPPORT.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+We use GitHub issues to track bugs and feature requests. If you need support help (i.e. with using InSpec), we recommend that if you are a:
+
+* Chef customer
+  * Please reach out to your Customer Success manager and they can assist you
```
Comment (mjingle) [inline SUPPORT.md]:
```suggestion
  * Please reach out to your Customer Success manager and they can assist you.
```

File: SUPPORT.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+We use GitHub issues to track bugs and feature requests. If you need support help (i.e. with using InSpec), we recommend that if you are a:
```
Comment (tas50) [inline SUPPORT.md]:
```suggestion
We use GitHub issues to track bugs and feature requests. If you need support help (i.e. with using Chef InSpec), we recommend that if you are a:
```

Comment (kekaichinose) [reply SUPPORT.md]: Went ahead and got this added to the latest commit

**Review by mjingle — APPROVED** [review]
tenor-68402456

**Pull request merged**